### ISSUE TITLE
v3.5.0 has an undeclared dependency on `git`

### ISSUE THREAD

**Issue by 72636c** [DataDog/serverless-plugin-datadog#216]
## Expected Behavior

Deployment succeeds after upgrading from v3.4.0 to minor version v3.5.0.

## Actual Behavior

Deployment fails with `spawn git ENOENT`.

## Steps to Reproduce the Problem

  1. Enter an environment without `git` on your PATH (e.g. the `node:16-alpine` image)
  1. Run `serverless deploy`

## Specifications

  - Serverless Framework version: 2.72.2
  - Datadog Serverless Plugin version: 3.5.0
  - Lambda function runtime (Python 3.7, Node 10, etc.): Node.js 14.x
  
## Stacktrace

```yaml
Serverless: Auto instrumenting functions with Datadog
--
  | Serverless: Adding Lambda Library Layers to functions
  | Serverless: Adding Datadog Lambda Extension Layer to functions
  | Serverless: Packaging service...
  | Serverless: Excluding development dependencies...
  | Serverless: Setting Datadog Forwarder
  | Serverless: Adding Plugin Version 3.5.0 tag
  | Serverless: Adding service and environment tags
  |  
  | Error ---------------------------------------------------
  |  
  | Error: Error: spawn git ENOENT
  | at Process.ChildProcess._handle.onexit (internal/child_process.js:274:19)
  | at onErrorNT (internal/child_process.js:469:16)
  | at processTicksAndRejections (internal/process/task_queues.js:82:21)
  | at GitExecutorChain.onFatalException (/workdir/node_modules/simple-git/src/lib/runners/git-executor-chain.js:61:87)
  | at GitExecutorChain.<anonymous> (/workdir/node_modules/simple-git/src/lib/runners/git-executor-chain.js:52:28)
  | at Generator.throw (<anonymous>)
  | at rejected (/workdir/node_modules/simple-git/src/lib/runners/git-executor-chain.js:6:65)
  |  
  | For debugging logs, run again after setting the "SLS_DEBUG=*" environment variable.
  |  
  | Get Support --------------------------------------------
  | Docs:          docs.serverless.com
  | Bugs:          github.com/serverless/serverless/issues
  | Issues:        forum.serverless.com
  |  
  | Your Environment Information ---------------------------
  | Operating System:          linux
  | Node Version:              14.18.3
  | Framework Version:         2.72.0 (local)
  | Plugin Version:            5.5.3
  | SDK Version:               4.3.0
  | Components Version:        3.18.1
  |  
```

**Comment by IvanTopolcic** [comment]
Thanks for the report! Looks like this is indeed something that we overlooked, I have a PR open [here](https://github.com/DataDog/serverless-plugin-datadog/pull/217) and will merge it in and deploy the fix after I've verified it's working.

**Comment by IvanTopolcic** [comment]
Fixed in [v3.5.1](https://github.com/DataDog/serverless-plugin-datadog/releases/tag/v3.5.1)
